### PR TITLE
Add metadata for HLS 1.10.0.0

### PR DESCRIPTION
**Pull request by wz1000**

### CODE REVIEW

File: ghcup-0.0.7.yaml [hunk 1]
```diff
@@ -4322,6 +4322,93 @@ ghcupDownloads:
...
+      viChangeLog: https://github.com/haskell/haskell-language-server/blob/master/ChangeLog.md
+      viPostInstall: *hls-post-install
+      viSourceDL:
+        dlUri: https://downloads.haskell.org/~hls/haskell-language-server-1.10.0.0/haskell-language-server-1.10.0.0-src.tar.gz
+        dlSubdir: haskell-language-server-1.10.0.0
+        dlHash: dc73a89b43ed7f5f1823220a6873a6f5033d6140dde847526599b9fb41b2ed08
+      viArch:
+        A_64:
+          Linux_Debian:
+            '< 10': &hls-191-64-deb9
```
Comment (hasufell) [inline ghcup-0.0.7.yaml:4337]: The yaml anchors are not correct

File: ghcup-0.0.7.yaml [hunk 2]
```diff
@@ -4322,6 +4322,93 @@ ghcupDownloads:
...
+          Linux_CentOS:
+            '( >= 7 && < 8 )': &hls-191-64-centos
+              dlUri: https://downloads.haskell.org/~hls/haskell-language-server-1.10.0.0/haskell-language-server-1.10.0.0-x86_64-linux-centos7.tar.xz
+              dlSubdir: haskell-language-server-1.10.0.0
+              dlHash: 873d20a068398a52671f710f4fb06736f109752c03c2177950887125ccde18ca
+            unknown_versioning: *hls-191-64-centos
+          Linux_RedHat:
+            unknown_versioning: *hls-191-64-centos
+          Linux_UnknownLinux:
+            unknown_versioning: *hls-191-64-fedora27
```
Comment (hasufell) [inline ghcup-0.0.7.yaml:4391]: unknown linux should have its own bindist

**Review by hasufell — COMMENTED** [review]
FreeBSD bindist is missing

**Comment by wz1000** [comment]
We are waiting for FreeBSD on 9.2.7 to be fixed.

**Pull request merged**